### PR TITLE
[webapi] Add webaudio.createAnalyser return value test

### PR DESCRIPTION
**Pull request by chunywang**
The value of channelCount get 2 not  except 1 on Crosswalk and Chrome browser desktop,
but it gets 1 on Firefox browser desktop.

Impacted TCs num(approved): New 2, Update 0, Delete 0
Unit test Platform: Crosswalk Project for Android 16.45.410.0
Unit test result summary: Pass 1, Fail 1, Blocked 0

### CODE REVIEW

**Comment by chunywang** [comment]
AudioContext.createAnalyser () return an AnalyserNode object, and it doesn't return zero. @zqzhang

File: webapi/tct-webaudio-w3c-tests/webaudio/audiocontext_createAnalyser_return_value.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<script src="./support/webaudio.js"></script>
+<div id="log"></div>
+<script>
+
+test(function() {
+  var context = new AudioContext();
+  var analyser = context.createAnalyser();
+  analyser.fftSize = 256;
+  assert_true(analyser instanceof AnalyserNode);
+  assert_not_equals(analyser, 0);
```
Comment (zqzhang) [inline webapi/tct-webaudio-w3c-tests/webaudio/audiocontext_createAnalyser_return_value.html]: Please also check something like default values at http://webaudio.github.io/web-audio-api/#idl-def-AnalyserNode:
```
    numberOfInputs  : 1
    numberOfOutputs : 1    Note that this output may be left unconnected.

    channelCount = 1;
    channelCountMode = "max";
    channelInterpretation = "speakers";

fftSize of type unsigned long,
The size of the FFT used for frequency-domain analysis. This must be a power of two in the range 32 to 32768, otherwise an IndexSizeError exception must be thrown. The default value is 2048. Note that large FFT sizes can be costly to compute.
frequencyBinCount of type unsigned long, readonly
Half the FFT size.
maxDecibels of type float,
The maximum power value in the scaling range for the FFT analysis data for conversion to unsigned byte values. The default value is -30. If the value of this attribute is set to a value less than or equal to minDecibels, an IndexSizeError exception must be thrown.
minDecibels of type float,
The minimum power value in the scaling range for the FFT analysis data for conversion to unsigned byte values. The default value is -100. If the value of this attribute is set to a value more than or equal to maxDecibels, an IndexSizeError exception must be thrown.
smoothingTimeConstant of type float,
A value from 0 -> 1 where 0 represents no time averaging with the last analysis frame. The default value is 0.8. If the value of this attribute is set to a value less than 0 or more than 1, an IndexSizeError exception must be thrown.
```

File: webapi/tct-webaudio-w3c-tests/webaudio/audiocontext_createAnalyser_return_value.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+}, "Check if AudioContext.createAnalyser() returns zero.");
```
Comment (zqzhang) [inline webapi/tct-webaudio-w3c-tests/webaudio/audiocontext_createAnalyser_return_value.html]: => Check that AudioContext.createAnalyser() returns an instance of AnalyserNode and gets default values of its members as specified

File: webapi/tct-webaudio-w3c-tests/webaudio/audiocontext_createAnalyser_return_value.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+<meta charset="utf-8">
+<title>WebAudio Test: audiocontext createAnalyser return value</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="author" title="Chunyan Wang" href="mailto:chunyanx.wang@intel.com">
+<link rel="help" href="http://webaudio.github.io/web-audio-api/#widl-AudioContext-createAnalyser-AnalyserNode">
```
Comment (zqzhang) [inline webapi/tct-webaudio-w3c-tests/webaudio/audiocontext_createAnalyser_return_value.html]: Please make another PR to update the spec links in this test suite, thanks.

File: webapi/tct-webaudio-w3c-tests/webaudio/audiocontext_createAnalyser_return_value.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+<script src="../resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+
+test(function() {
+  var context = new AudioContext();
+  var analyser = context.createAnalyser();
+
+  assert_true(analyser instanceof AnalyserNode);
+  assert_not_equals(analyser, 0);
```
Comment (haoxli) [inline webapi/tct-webaudio-w3c-tests/webaudio/audiocontext_createAnalyser_return_value.html]: @zqzhang @chunywang I think there is no need to test the return value, we have verified the return value is Object AnalyserNode in type checking. And assert `analyser != 0` just to meet the bug requirement, which we cannot reproduce, but no statement in spec.

The default value checking for AnalyserNode attribute can been added in audiocontext_createAnalyser_base or create new tests for AnalyserNode interface.

Comment (zqzhang) [inline webapi/tct-webaudio-w3c-tests/webaudio/audiocontext_createAnalyser_return_value.html]: make sense

Comment (zqzhang) [inline webapi/tct-webaudio-w3c-tests/webaudio/audiocontext_createAnalyser_return_value.html]: Any update? @chunywang 

If not sure how to update, please ask @haoxli

Comment (chunywang) [inline webapi/tct-webaudio-w3c-tests/webaudio/audiocontext_createAnalyser_return_value.html]: I have update the test, please review. @zqzhang
